Resources.py

```
@@ -1,497 +1,497 @@
-		try:
-			with open(filePath, "w", encoding="utf-8") as json_file:
-				json.dump(dataDict, json_file, ensure_ascii=False, indent=2)
-			cls.__logger.info(f"Data successfully saved to {filePath}")
-		except Exception:
+﻿import gzip
+import json
+import logging
+import os
+import shutil
+import threading
+from Resources import *
```

Ruff (0.14.8)

`from Resources import *` used; unable to detect undefined names

(F403)